src/NexusMods.App.UI/LeftMenu/Items/LeftMenuItemViewModel.cs

```diff
@@ -17,10 +18,13 @@ public class LeftMenuItemViewModel : AViewModel<ILeftMenuItemViewModel>, ILeftMe
 {
     public StringComponent Text { get; init; } = new("");
     [Reactive] public IconValue Icon { get; set; } = new();
+    [Reactive] public string ToolTipText { get; private set; }
     [Reactive] public bool IsActive { get; private set; }
     [Reactive] public bool IsSelected { get; private set; }
     public ReactiveCommand<NavigationInformation, Unit> NavigateCommand { get; }
 
+    public StringComponent? ToolTip { get; init; }
```

Comment:
This doesn't have to be nullable. You can just provide a default value:

```csharp
ToolTip { get; init; } = new(value: string.Empty);
```

This way you don't need an extra `ToolTipText` property and can just use this.

Reply:
I made it nullable to be able to know whether it was set or not, if not set it is meant to fallback to showing the Text as the tooltip. 
I think it would be more messy to try to make the fallback work using a default value.

The ToolTipText property also makes things easier since I need to call `ToolTip.SetTip()` in the code behind, rather than doing a binding.